hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/tools/TestGetConf.java

```diff
@@ -484,10 +483,10 @@ public void testGetJournalNodes() throws Exception {
     conf.clear();
   }
 
-  /*
-   ** Test for unknown journal node host exception.
+  /**
+   * Test handling of unresolvable journal node hosts.  They are still configured assuming that
+   * they will be resolvable in the future.
   */
-  @Test(expected = UnknownHostException.class, timeout = 10000)
```

Comment:
Do you mean for this test method to no longer be executed?

Reply:
Corrected.